client/android/src/org/amnezia/vpn/VPNService.kt

```diff
@@ -802,6 +802,8 @@ Log.i(tag, "shadowsocksConfig: profile.remotePort" + profile.remotePort)
                     }
             if (!profile.bypass) builder.addAllowedApplication(me)
         }
+        val me = packageName
+        builder.addDisallowedApplication(me)
```

Comment:
This can be written in one line

Reply:
yep, in general work with profile.bypass can be better.